pkg/handler/iam_handler.go

```diff
@@ -116,7 +116,17 @@ func (h *IAMHandler) handlePolicy(ctx context.Context, p *v1alpha1.ResourcePolic
 	var np *iampb.Policy
 	if err := retry.Do(ctx, h.retry, func(ctx context.Context) error {
 		// Get current IAM policy.
-		cp, err := iamC.GetIamPolicy(ctx, &iampb.GetIamPolicyRequest{Resource: p.Resource})
+		getIAMPolicyRequest := &iampb.GetIamPolicyRequest{
+			Resource: p.Resource,
+			// Set required policy version to 3 to support conditional IAM bindings
+			// in the requested policy.
+			// Note that if the requested policy does not contain conditional IAM
+			// bindings it will return the policy as is, which is version 1.
```

Comment:
Thanks! Please add a link to official doc.

Reply:
Done, resolving!